pmoves/services/cast-tts-gateway/service.py

```
@@ -0,0 +1,2074 @@
+                        async with httpx.AsyncClient(timeout=120.0) as client:
+                            response = await client.post(
+                                f"{ULTIMATE_TTS_URL}/api/predict",
+                                json={"data": [text, voice_model, speed, pitch, speed]},
+                            )
+                            response.raise_for_status()
+                            result_data = response.json()
+
+                            if "data" in result_data and len(result_data["data"]) > 0:
+                                audio_path = result_data["data"][0]
+
+                    except Exception as e:
```

Ruff (0.15.5)

[warning] Do not catch blind exception: `Exception`

(BLE001)